gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -504,13 +671,27 @@ properties:
     $ref:
       - "_terms.yaml#/classification_of_tumor/common"
     enum:
-      - primary
       - metastasis
-      - recurrence
       - other
+      - Premalignant
```

Comment:
Decided to leave as upper-case as the normalization on the portal will make this all look lower case. We also want to start migrating towards title-case in the dictionary.